debian/docs/one-import-bcq.1

```diff
@@ -0,0 +1,56 @@
+.TH ONE-IMPORT-BCQ "1" "June 2021" "one-import-bcq version 1.16.1" "User Commands"
+.SH NAME
+one-import-bcq \- convert TensorFlow with BCQ to circle
+.SH DESCRIPTION
+usage: one\-import\-bcq [\-h] [\-v] [\-C CONFIG] [\-\-v1 | \fB\-\-v2]\fR [\-i INPUT_PATH]
+.br
+[\-o OUTPUT_PATH] [\-I INPUT_ARRAYS] [\-s INPUT_SHAPES] [\-O OUTPUT_ARRAYS]
+.PP
+\fBone\-import\-bcq\fR is a command line tool to convert TensorFlow with BCQ to circle.
+.SH OPTIONS
+.TP
+\fB\-h\fR, \fB\-\-help\fR
+show this help message and exit
+.TP
+\fB\-v\fR, \fB\-\-version\fR
+show program's version number and exit
+.TP
+\fB\-C\fR CONFIG, \fB\-\-config\fR CONFIG
+run with configuation file
+.TP
+\fB\-\-v1\fR
+use TensorFlow Lite Converter 1.x
```

Comment:
`TensorFlow Lite` -> `TensorFlow lite`

Reply:
@seanshpark Most of help messages are from its driver's. Should it be changed, too?

Reply:
`https://www.tensorflow.org/lite?hl=en` now uses `Lite`... some time ago, I remember site used `lite` ...
Let's keep this as-is -_-;;;